src/Avalonia.Base/Rendering/Composition/Drawing/CompositionDrawListSceneBrushContent.cs

```diff
@@ -18,20 +18,37 @@ public CompositionDrawListSceneBrushContent(ImmutableTileBrush brush, Compositio
 
     public ITileBrush Brush { get; }
     public Rect Rect { get; }
-    
+
     public double Opacity => Brush.Opacity;
     public ITransform? Transform => Brush.Transform;
     public RelativePoint TransformOrigin => Brush.TransformOrigin;
-    
+
     public void Dispose() => _drawList.Dispose();
 
     public void Render(IDrawingContextImpl context, Matrix? transform)
     {
-        if (transform.HasValue)
-            _drawList.Render(context, transform.Value);
-        else
-            _drawList.Render(context);
+        var previousRenderOptions = context.RenderOptions;
+
+        switch (previousRenderOptions.TextRenderingMode)
+        {
+            case TextRenderingMode.Unspecified:
+            case TextRenderingMode.SubpixelAntialias:
+                context.RenderOptions = previousRenderOptions with { TextRenderingMode = TextRenderingMode.Antialias };
```

Comment:
What happens if some control down the tree sets `SubpixelAntialias`? I think we should disable it on the DrawingContext level.

Reply:
Note that we do set `DisableTextLcdRendering` here
https://github.com/AvaloniaUI/Avalonia/blob/ab4436ca5da48f151653d6ab59d737ad79dc6f04/src/Skia/Avalonia.Skia/PictureRenderTarget.cs#L38-L45

Reply:
It seems that `DisableTextLcdRendering` is being ignored for some reason. Any idea why? It was supposed to forcibly disable subpixel rendering for transparent render targets

Reply:
Well, how are you planning to fix child controls that re-enable the subpixel rendering then? 
The fact that subpixel rendering isn't supported should be a property of the current render target, if some render target claims that it doesn't support it while it should, we should fix that render target instead of just ignoring that flag.

Reply:
It is probably easier to disable ClearType when the Window is transparent. In general for all non opaque surfaces.